lib/actual_db_schema/migration.rb

```diff
@@ -78,6 +58,12 @@ def rollback(version, database)
       end
     end
 
+    def rollback_all
+      MigrationContext.instance.each do |context|
+        context.rollback_branches(manual_mode: false)
```

Comment:
```suggestion
        context.rollback_branches
```

this is a default option